android/src/main/java/dev/openfeature/sdk/FeatureProvider.kt

```diff
@@ -27,4 +27,16 @@ interface FeatureProvider : EventObserver, ProviderStatus {
     fun getIntegerEvaluation(key: String, defaultValue: Int, context: EvaluationContext?): ProviderEvaluation<Int>
     fun getDoubleEvaluation(key: String, defaultValue: Double, context: EvaluationContext?): ProviderEvaluation<Double>
     fun getObjectEvaluation(key: String, defaultValue: Value, context: EvaluationContext?): ProviderEvaluation<Value>
+
+    /**
+     * Feature provider implementations can opt in for to support Tracking by implementing this method.
+     *
+     * Performs tracking of a particular action or application state.
+     *
+     * @param trackingEventName Event name to track
+     * @param context   Evaluation context used in flag evaluation (Optional)
+     * @param details   Data pertinent to a particular tracking event (Optional)
+     */
+    fun track(trackingEventName: String, context: EvaluationContext?, details: TrackingEventDetails?) {
```

Comment:
Can you please confirm that this can be optionally implemented by a provider?

Reply:
left a comment in the function body. the interface can have a default implementation making implementing it optional. (you can see that this works from the different Providers in the test suite)